libraries/AP_HAL/SIMState.cpp

```diff
@@ -163,6 +163,9 @@ void SIMState::fdm_input_local(void)
     if (leddarone != nullptr) {
         leddarone->update(sitl_model->rangefinder_range());
     }
+    if (rds02uf != nullptr) {
```

Comment:
These AP_HAL_SIMState changes should be moved to a separate commit prefixed with "AP_HAL:"